src/pages/blog/video-banner/_VideoCarousel.js

```diff
@@ -0,0 +1,210 @@
+                                <ItemsMainWrapper
+                                    jc="flex-start"
+                                    key={index}
+                                    onClick={handleOpenVideo}
+                                >
+                                    <ImgWrapper width="139px">
+                                        <ImgDiv src={item.img_url} alt={item.image} />
+                                        <PlayerIconWrapper absolute ai="center">
+                                            <IconDiv>
+                                                <PlayerIcon src={PlayIcon} />
+                                            </IconDiv>
+                                        </PlayerIconWrapper>
+                                    </ImgWrapper>
+                                    <Flex direction="column" ml="8px" width="180px">
+                                        <Header as="p" type="paragraph-1" color="white" mb="4px">
+                                            {item.title}
+                                        </Header>
+                                        <SmallDetailsWrapper
+                                            height="24px"
+                                            jc="flex-start"
+                                            ai="center"
+                                        >
+                                            <Header
+                                                as="p"
+                                                type="paragraph-2"
+                                                weight="normal"
+                                                color="grey-17"
+                                                width="auto"
+                                            >
+                                                {item.date}
+                                            </Header>
+                                            <StyledDot />
+                                            <Header
+                                                as="p"
+                                                type="paragraph-2"
+                                                weight="normal"
+                                                color="grey-17"
+                                                width="auto"
+                                            >
+                                                {item.duration}
+                                            </Header>
+                                        </SmallDetailsWrapper>
+                                    </Flex>
+                                </ItemsMainWrapper>
+                            )
+                        })}
+                    </CustomCarousel>
+                </Flex>
+                {is_mobile && (
+                    <SeeMoreBtnMobile to="/">
+                        <Header type="paragraph-2" color="white" align="center">
+                            See all videos
```

Comment:
What will happen, if someone click this button?

Reply:
It should bring the user to `All videos` page I believe..Correct @sean-binary ?

Reply:
That's right. Should link to `/blog/videos`